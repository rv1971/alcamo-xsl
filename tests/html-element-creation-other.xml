<?xml version="1.0" encoding="UTF-8"?>

<?xml-stylesheet href="#xslt" type="text/xml"?>

<a:doc
    xmlns:a="tag:rv1971@web.de,2021:alcamo-xsl#"
    xmlns:dc="http://purl.org/dc/terms/"
    xmlns:owl="http://www.w3.org/2002/07/owl#"
    xmlns:rdfs="http://www.w3.org/2000/01/rdf-schema#">
  <xsl:stylesheet
      xmlns="http://www.w3.org/1999/xhtml"
      xmlns:dc="http://purl.org/dc/terms/"
      xmlns:owl="http://www.w3.org/2002/07/owl#"
      xmlns:rdfs="http://www.w3.org/2000/01/rdf-schema#"
      xmlns:xsl="http://www.w3.org/1999/XSL/Transform"
      xml:id="xslt"
      version="1.0"
      exclude-result-prefixes="dc owl">
    <xsl:import href="../xsl/html.xsl"/>

    <xsl:template match="/">
      <a:result>
        <xsl:apply-templates select="/a:doc/a:data/*"/>
     </a:result>
    </xsl:template>

    <xsl:template match="a:*|dc:*|owl:*|rdfs:*">
      <xsl:apply-templates select="." mode="a:li"/>
      <xsl:apply-templates select="." mode="a:p"/>
    </xsl:template>
  </xsl:stylesheet>

  <a:data>
    <a:item>foo</a:item>
    <dc:creator>alice@example.org</dc:creator>
    <dc:rightsHolder>mailto:bob@example.org</dc:rightsHolder>
    <owl:versionInfo>0.42.1</owl:versionInfo>
    <rdfs:comment>Lorem ipsum.</rdfs:comment>
  </a:data>
</a:doc>
