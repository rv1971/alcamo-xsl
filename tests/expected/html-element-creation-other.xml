<?xml version="1.0"?>
<a:result xmlns:a="tag:rv1971@web.de,2021:alcamo-xsl#" xmlns="http://www.w3.org/1999/xhtml" xmlns:rdfs="http://www.w3.org/2000/01/rdf-schema#">
  <p>foo</p>
  <p>
    <a href="mailto:alice@example.org">alice</a>
  </p>
  <p>
    <a href="mailto:bob@example.org">bob</a>
  </p>
  <p>0.42.1</p>
  <p>Lorem ipsum.</p>
</a:result>
